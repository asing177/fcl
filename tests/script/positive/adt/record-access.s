type T
  = C1(int f1, bool f2, text f3)
  | C2(int f1, bool f2)
  | C3(int f1);

int x;

@initial
go(T t) {
  x = t.f1;
  terminate()
}
