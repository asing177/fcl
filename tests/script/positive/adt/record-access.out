 
Method arguments and effects:
go: (T) -> {write via terminate, write x}